cmd/addoperator.go

```diff
@@ -216,9 +218,15 @@ func (p *AddOperatorParams) PostInteractive(ctx ActionCtx) error {
 func (p *AddOperatorParams) Validate(ctx ActionCtx) error {
 	var err error
 	if p.token != "" {
+		if p.sysAcc {
+			return fmt.Errorf("importing an operator is not compatable with system account generation")
```

Comment:
```suggestion
			return fmt.Errorf("importing an operator is not compatible with system account generation")
```

Reply:
fixed

Reply:
dang, didn't notice that commit suggestion button